### PR TITLE
Need npm deploy to build customized skill

### PR DESCRIPTION
**Pull request by rgstephens**
Thanks for this nice TypeScript starter app. Needed to do `npm deploy` first time to do the build. For some reason on my system I had to do `npm run-script deploy`. Not sure why npm couldn't pick up that this was a script command.

### CODE REVIEW

File: .history/README_20180813094516.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+[![Build Status](https://travis-ci.org/Xzya/alexa-typescript-starter.svg?branch=master)](https://travis-ci.org/Xzya/alexa-typescript-starter)
```
Comment (Xzya) [inline .history/README_20180813094516.md]: Can you delete this file?

File: .history/README_20180813102211.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+[![Build Status](https://travis-ci.org/Xzya/alexa-typescript-starter.svg?branch=master)](https://travis-ci.org/Xzya/alexa-typescript-starter)
+[![codecov](https://codecov.io/gh/Xzya/alexa-typescript-starter/branch/master/graph/badge.svg)](https://codecov.io/gh/Xzya/alexa-typescript-starter)
```
Comment (Xzya) [inline .history/README_20180813102211.md]: Also this

File: README.md [hunk 3]
```diff
@@ -207,7 +207,7 @@ Taken from [the official hello world project](https://github.com/alexa/skill-sam
 4. Remember to re-deploy your skill and Lambda function for your changes to take effect.
 
 	```bash
-	$ ask deploy
+	$ npm run-script deploy
```
Comment (Xzya) [inline README.md]: `run-script` and `run` both do the same thing, but `run` is more commonly used.

Could you change this to `$ npm run deploy` please?

**Review by Xzya — CHANGES_REQUESTED** [review]
Hello,

Thanks your feedback!

`ask deploy` and `npm run deploy` are two separate commands.

`ask deploy` uses the [ASK-CLI](https://developer.amazon.com/docs/smapi/quick-start-alexa-skills-kit-command-line-interface.html) to make a deployment, while `npm run deploy` will run the script inside `package.json` which will first build the project and then run `ask deploy`, as seen [here](https://github.com/Xzya/alexa-typescript-starter/blob/master/package.json#L9).

Also `npm deploy` is an invalid command as it requires you to add `run` or `run-script`.

That being said, I think you are right. Changing the command inside the README should be better.

Just a few minor changes:

**Comment by rgstephens** [comment]
In the process of trying to clean up the pull request, I ended up making some newbie git mistakes but I think the latest commit is correct.

**Review by Xzya — APPROVED** [review]

**Pull request merged**